src/index.ts

```diff
@@ -0,0 +1,65 @@
+import { Editor } from 'grapesjs'
+import { KeymapsDialogManager } from './KeymapsDialogManager'
+
+const cmdKeymapsDialog = 'open-keymaps-dialog'
+
+export default (editor: Editor, opts = {}): void => {
+  const options = {
+    ...{
+      longPressKey: 'shift',
+      longPressDuration: 800,
+      shortcut: 'shift+k',
+      css: null
+    }, ...opts
+  }
+
+  const manager = new KeymapsDialogManager(editor, options)
+
+  // Command
+  editor.Commands.add(cmdKeymapsDialog, {
```

Comment:
good to have a command :+1: 
we will need a button somewhere in the UI in silex